client/src/components/BookSitter/BookSitter.tsx

```diff
@@ -1,64 +1,82 @@
-import {
-  Typography,
-  Grid,
-  CircularProgress,
-  Paper,
-  Card,
-  CardMedia,
-  CardContent,
-  Box,
-  TextField,
-  Button,
-} from '@material-ui/core';
-import { Rating } from '@material-ui/lab';
+import { Typography, Grid, Card, Button } from '@material-ui/core';
+import Rating from '@material-ui/lab/Rating';
+import Alert from '@material-ui/lab/Alert';
+import { useLocation } from 'react-router-dom';
 import { useAuth } from '../../context/useAuthContext';
-import house from '../../Images/house.jpg';
 import useStyles from './useStyles';
+import { useState } from 'react';
+import { KeyboardDatePicker, KeyboardTimePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import createRequests from '../../helpers/APICalls/createRequest';
 
 const BookSitter = (): JSX.Element => {
   const classes = useStyles();
   const { loggedInUser } = useAuth();
+  const [date, handleDate] = useState<Date | null>(new Date());
+  const [startTime, handleStartTime] = useState<Date | null>(new Date());
+  const [endTime, handleEndTime] = useState<Date | null>(new Date());
+  const [isAlert, setIsAlert] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const location = useLocation();
+  const sendRequest = () => {
+    createRequests(startTime, endTime, date, '60d4994ec30c55353c3e5d93', '60d4996fc30c55353c3e5d94').then((res) => {
+      if (res.error) {
+        setIsError(true);
+      } else if (res) {
+        setIsAlert(true);
```

Comment:
set error to false here :)

Reply:
instead of using boolean, this time I added string so initially it will be empty and no alert will be displayed and if successful setting it to "true" will give success alert and false will give "error" alert